include/hal/library/cryptlib.h

```diff
@@ -1942,42 +1936,21 @@ bool libspdm_sm2_dsa_verify(const void *sm2_context, size_t hash_nid,
                             const uint8_t *signature, size_t sig_size);
 
 /*=====================================================================================
- *    Pseudo-Random Generation Primitive
+ *    Random Number Generation Primitive
  *=====================================================================================*/
 
 /**
- * Sets up the seed value for the pseudorandom number generator.
- *
- * This function sets up the seed value for the pseudorandom number generator.
- * If seed is not NULL, then the seed passed in is used.
- * If seed is NULL, then default seed is used.
- * If this interface is not supported, then return false.
- *
- * @param[in]  seed      Pointer to seed value.
- *                      If NULL, default seed is used.
- * @param[in]  seed_size  size of seed value.
- *                      If seed is NULL, this parameter is ignored.
- *
- * @retval true   Pseudorandom number generator has enough entropy for random generation.
- * @retval false  Pseudorandom number generator does not have enough entropy for random generation.
- * @retval false  This interface is not supported.
- *
- **/
-bool libspdm_random_seed(const uint8_t *seed, size_t seed_size);
-
-/**
- * Generates a pseudorandom byte stream of the specified size.
+ * Generates a random byte stream of the specified size.
```

Comment:
can we add some comment, such as the library provide or integrator should initialize the random seed to ensure the randomness.

Reply:
It is done.